src/psyclone/tests/nemo/transformations/openacc/data_directive_test.py

```diff
@@ -205,7 +205,7 @@ def test_array_syntax():
     schedule = psy.invokes.get('tra_ldf_iso').schedule
```

Comment:
Not a change here but could you please check ...

C: 45, 0: third party import "from psyclone.parse.algorithm import parse" should be placed before "from fparser.common.readfortran import FortranStringReader" (wrong-import-order)
C: 46, 0: third party import "from psyclone.psyGen import PSyFactory, TransInfo, InternalError" should be placed before "from fparser.common.readfortran import FortranStringReader" (wrong-import-order)
C: 47, 0: third party import "from psyclone.transformations import TransformationError" should be placed before "from fparser.common.readfortran import FortranStringReader" (wrong-import-order)

Reply:
My pylint appears to be completely ambivalent about this so I've changed it :-)